p2p/peer.py

```diff
@@ -754,7 +760,8 @@ def add_peer(self, peer):
             await self._connect_to_nodes(self._get_random_bootnode())
 
     def _get_random_bootnode(self) -> Generator[Node, None, None]:
-        yield random.choice(self.discovery.bootstrap_nodes)
+        if self.discovery.bootstrap_nodes:
+            yield random.choice(self.discovery.bootstrap_nodes)
```

Comment:
Maybe log a warning when there are no bootstrap nodes, as that will probably lead to us not finding any nodes to connect to. Or maybe we should make `bootstrap_nodes` required

Reply:
I'd like to add warning message because sometimes some nodes really don't have bootstrap nodes for testing and testnet.